### ISSUE TITLE
getting nowhere

### ISSUE THREAD

**Issue by ghost** [renard/o-blog#151]
I'm not sure if this project is still alive but nothing seems to work.

I installed from elpa and got nothing workable.

I checked out from git and followed instructions.
From sample.org running _M-x o-blog-publish_ yielded:
**eieio-defclass: Given parent class ob:backend is not a class**

I tried following the instructions on _http://renard.github.io/o-blog/bug-report.html_
Emacs ended up in the scratch buffer and Messages was:
**command-line-1: Cannot open load file: No such file or directory, .emacs.d/o-blog/sample-init.el**

Please advise.

Emacs 24.5.1
Org 8.2.10

**Comment by cviebig** [comment]
I opted to clone the git repository directly into `.emacs.d` and  adding the following to `init.el`:

	(add-to-list 'load-path "~/.emacs.d/o-blog/lisp")
	(require 'o-blog)

**Comment by jtgd** [comment]
Thanks cviebig, adding the */lisp* to the path did change it's behavior. Now the error is:
**progn: Opening input file: No such file or directory, /home/jay/.emacs.d/o-blog/Makefile**
and there is indeed a reference to a *Makefile* in the sample.org file which is not present.
I'll assume it was an oversight by whoever wrote the sample.

But at least it didn't fail like before. I think I will proceed to try to generate my own file and see where that gets me.  Thanks again for the tip.

**Comment by cviebig** [comment]
@jtgd Weird. No, I did not need to run `make`. Did you install `pandoc`? [see here.](http://renard.github.io/o-blog-v2.6/index.html) Also checkout the v2.6 branch. The most recent code is not in `master`.

**Comment by cviebig** [comment]
@jtgd Also install `htmlize` (as in #145) package if you have not already.

**Comment by fxfactorial** [comment]
I also have this problem. Frankly it seems this project was abandoned.

**Comment by drtutut** [comment]
emacs 24.3.1, org-mode 8.3.1, cloned git repository of o-blog and switched to branch 2.6

I face the same problems. It seems that the examples are not up to date with the version (at least the org-mode ones). When I try to publish the sample org file, emacs complains : `unable to create link "Creating a blog"`, but as no output is generated, I can hardly track down the problem.

I also tried to generate a minimal org-mode file. The result is visually bad as the styles are not copied in the publishing directory, although I have set `#+TEMPLATE_DIR:` and `#+STYLE_DIR:`

All of this makes me thinks that the developer choose to concentrate on markdown format.
